general-concepts/ebuild-revisions/text.xml

```diff
@@ -52,8 +52,8 @@ of thumb could be used as a guideline:
     etc.) and it would not be propagated using other means, then
     the ebuild should be renamed to a new revision. If the package has
     stable keywords, they should be moved to the new revision without
-    dropping. To commit the ebuild, <c>repoman commit
-    --straight-to-stable</c> option should be used.
+    dropping. When using <c>repoman commit</c>, the <c>--straight-to-stable</c>
+    option can be used to disable stable keyword warnings.
```

Comment:
Sorry, but what was wrong with the old wording?
Why are we going to discourage repoman now?